common/djangoapps/request_cache/middleware.py

```diff
@@ -108,7 +109,7 @@ def func_call_cache_key(func, *args, **kwargs):
     the function's name, and a stringified list of arguments
     and a query string-style stringified list of keyword arguments.
     """
-    converted_args = map(str, args)
-    converted_kwargs = map(str, reduce(list.__add__, map(list, sorted(kwargs.iteritems())), []))
+    converted_args = map(force_text, args)
+    converted_kwargs = map(force_text, reduce(list.__add__, map(list, sorted(kwargs.iteritems())), []))
```

Comment:
Cool. I've confirmed with Performance team and using force_text should't have performance impact.